kcidb/orm/argparse.py

```diff
@@ -0,0 +1,94 @@
+"""
+KCIDB object-relational mapping (ORM)-management
+command-line argument parsing
+"""
+
+import argparse
+import kcidb.misc
+import kcidb.argparse
+from kcidb.orm.query import Pattern
+
+# We'll get to it, pylint: disable=too-many-lines
+
+
+class PatternHelpAction(argparse.Action):
+    """Argparse action outputting pattern string help and exiting."""
+    def __init__(self,
+                 option_strings,
+                 dest=argparse.SUPPRESS,
+                 default=argparse.SUPPRESS,
+                 help=None):
+        super().__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            nargs=0,
+            help=help)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        print(
+            Pattern.STRING_DOC +
+            "\n" +
+            "NOTE: Specifying object ID lists separately is not "
+            "supported using\n"
+            "      command-line tools. "
+            "Only inline ID lists are supported.\n"
+        )
+        parser.exit()
+
+
+def add_args(parser):
+    """
+    Add common ORM arguments to an argument parser.
+
+    Args:
+        The parser to add arguments to.
+    """
+    parser.add_argument(
+        "pattern_strings",
+        nargs='*',
```

Comment:
Please revert this change of double quotes to single quotes:
```suggestion
        nargs="*",
```
It's unrelated to this commit.